h2integrate/converters/nitrogen/simple_ASU.py

```diff
@@ -0,0 +1,328 @@
+    recommended to set the parameter `efficiency_kWh_pr_kg_N2` to 0.119.
+    To represent a pressure swing absorption ASU, it is
+    recommended to set the parameter `efficiency_kWh_pr_kg_N2` to 0.29.
+
+    Attributes:
+        size_from_N2_demand (bool): if True, size the system based on some input demand. If False,
+            size the system from user input (`rated_N2_kg_pr_hr` or `ASU_rated_power_kW`).
+        rated_N2_kg_pr_hr (float | None): Rated capacity of ASU in kg-N2/hour. Only required if
+            `size_from_N2_demand` is False and ASU_rated_power_kW is not input.
+        ASU_rated_power_kW (float | None): Rated capacity of ASU in kg-N2/hour. Only required if
+            `size_from_N2_demand` is False and `rated_N2_kg_pr_hr` is not input.
+        efficiency_kWh_pr_kg_N2 (float): efficiency of the ASU in kWh/kg-N2, defaults to 0.29.
+            Should be between 0.1 and 0.5. Some reference efficiencies are::
+                - 0.29 for pressure swing absorption
+                - 0.119 for cryogenic
+        N2_fraction_in_air (float, optional): nitrogen content of input air stream as mole fraction.
+            Defaults to 0.7811.
+        O2_fraction_in_air (float, optional): oxygen content of input air stream as mole fraction.
+            Defaults to 0.2096.
+        Ar_fraction_in_air (float, optional): argon content of input air stream as mole fraction.
+            Defaults to 0.0093.
+    """
+
+    size_from_N2_demand: bool = field()
+    rated_N2_kg_pr_hr: float | None = field(default=None)
+    ASU_rated_power_kW: float | None = field(default=None)
+
+    N2_fraction_in_air: float = field(default=0.7811, validator=range_val(0, 1))
+    O2_fraction_in_air: float = field(default=0.2096, validator=range_val(0, 1))
+    Ar_fraction_in_air: float = field(default=0.0093, validator=range_val(0, 1))
+    efficiency_kWh_pr_kg_N2: float = field(default=0.29, validator=range_val(0.10, 0.50))
+    # 0.29 is efficiency of pressure swing absorption
+    # 0.119 is efficiency of cryogenic
+
+    def __attrs_post_init__(self):
+        if self.size_from_N2_demand:
+            return
+        if self.rated_N2_kg_pr_hr is None and self.ASU_rated_power_kW is None:
+            msg = (
+                "Either rated_N2_kg_pr_hr or ASU_rated_power_kW must be input if "
+                "size_from_N2_demand is False"
+            )
+            raise ValueError(msg)
+        else:
+            return
+
+
+class SimpleASUPerformanceModel(om.ExplicitComponent):
+    """Simple linear converter to model nitrogen production from an
+    Air Separation Unit.
+    """
+
+    def initialize(self):
+        self.options.declare("plant_config", types=dict)
+        self.options.declare("tech_config", types=dict)
+        self.options.declare("driver_config", types=dict)
+        self.N2_molar_mass = 28.0134  # grams/mol
+        self.O2_molar_mass = 15.999  # grams/mol
+        self.Ar_molar_mass = 39.948  # grams/mol
```

Comment:
This N2 value is for two nitrogen atoms; the O2 value is for just one. I think the O2 value should be doubled? But I want to make sure it's consistent and expected.